website/sidebarsIntegrations.js

```diff
@@ -103,6 +103,7 @@ module.exports = {
                         "services/sonarr/index",
                         "services/tautulli/index",
                         "services/weblate/index",
+                        "services/sonar-qube/index",
```

Comment:
```suggestion
                        "services/sonar-qube/index",
                        "services/sonarr/index",
                        "services/tautulli/index",
                        "services/weblate/index",
```
We're trying to keep this list sorted. Sorry about this, I should have mentioned it earlier.